src/backend/parser/cypher_clause.c

```diff
@@ -458,6 +468,69 @@ static Query *transform_cypher_delete(cypher_parsestate *cpstate,
     return query;
 }
 
+static Query *transform_cypher_unwind(cypher_parsestate *cpstate,
+                                      cypher_clause *clause)
+{
+    ParseState *pstate = (ParseState *) cpstate;
+    cypher_unwind *self = (cypher_unwind *) clause->self;
+
```

Comment:
Please group all of the variable definitions together, unless there are comments in between